src/Scaffold/Console/model/create_table.stub

```diff
@@ -1,15 +1,14 @@
 <?php namespace {{studly_author}}\{{studly_plugin}}\Updates;
 
+use Illuminate\Database\Schema\Blueprint;
 use Schema;
 use October\Rain\Database\Updates\Migration;
 
 class Create{{studly_plural_name}}Table extends Migration
 {
-
     public function up()
     {
-        Schema::create('{{lower_author}}_{{lower_plugin}}_{{snake_plural_name}}', function($table)
-        {
+        Schema::create('{{lower_author}}_{{lower_plugin}}_{{snake_plural_name}}', function (Blueprint $table) {
```

Comment:
Blueprints have been added to the migration stub as part of #212 which switches over to `October\Rain\Schema\Blueprint`